gef.py

```diff
@@ -795,22 +795,42 @@ def __str__(self):
 
 
 class GlibcChunk:
-    """Glibc chunk class.
+    """Glibc chunk class. The default behavior (from_base=False) is to interpret the data starting at the memory
+    address pointed to as the chunk data. Setting from_base to True instead treats that data as the chunk header.
```

Comment:
Can you add a reference to the Issue number for explaining the `allow_unaligned` ?